source/h2_decoder.c

```diff
@@ -89,19 +91,26 @@ DEFINE_STATE(priority_block, 5);
 /* Frame-specific states */
 DEFINE_STATE(frame_data, 0);
 DEFINE_STATE(frame_headers, 0);
-DEFINE_STATE(frame_priority, 5);
+DEFINE_STATE(frame_priority, 0);
```

Comment:
Isn't this size the size consumed? Or is it the size extracted?

Reply:
This state itself doesn't actually consume any data, `state_priority_block` does. See below, on line 565:
```c
static int s_state_fn_frame_priority(struct aws_h2_decoder *decoder, struct aws_byte_cursor *input) {
    (void)input;

    /* No data to process here, we're done! */
    s_decoder_reset_state(decoder);

    return AWS_OP_SUCCESS;
}
```